src/components/read-along-component/read-along.tsx

```diff
@@ -44,22 +48,36 @@ export class ReadAlongComponent {
    * Overlay
    * This is an SVG overlay to place over the progress bar
    */
-  @Prop() svg_overlay: string;
+  @Prop() svgOverlay: string;
 
   /**
-  * Theme to use: ['light', 'dark'] defaults to 'dark'
-  */
-  @Prop({ mutable: true }) theme: string = 'light';
+   * Theme to use: ['light', 'dark'] defaults to 'dark'
+   */
+  @Prop({mutable: true}) theme: string = 'light';
 
   /**
    * Language
    */
   @Prop() language: string = 'eng';
```

Comment:
```suggestion
  /**
   * Language of the interface. Options are "en" for English and "fr" for French.
   */
  @Prop() language: "en" | "fr" = 'en';
```

Reply:
changed it to the interface language type you defined below